index.coffee

```diff
@@ -86,6 +89,8 @@ class ProgressReporter
 				if status is 'Pulling fs layer'
 					downloadProgressTracker.addLayer(id)
 					extractionProgressTracker.addLayer(id)
+				else if status is 'Ready to download'
+					downloadProgressTracker.linkLayer(extractionProgressTracker, id)
```

Comment:
Can you add a comment about why the `'Ready to download'` message is special and requires us to link the download and extraction steps please